gateway.c

```diff
@@ -604,6 +606,8 @@ void LoadConfigFile()
 	ReadString(fp, "ftpPassword", Config.ftpPassword, sizeof(Config.ftpPassword), 0);
 	ReadString(fp, "ftpFolder", Config.ftpFolder, sizeof(Config.ftpFolder), 0);	
 
+	ReadBoolean(fp, "EnableKML", 0, &Config.EnableKML);

```

Comment:
I agree that an EnableKML bolean seems consistent with the other features and perhaps LogTelemetry/LogKML/LogRSSI boleans if/when the ability to log these values is added in future?